store/jdbc/src/main/java/com/github/mizool/typemapping/store/jdbc/JdbcBooleanValueLoadStrategy.java

```diff
@@ -5,16 +5,17 @@
 import java.sql.Types;
 
 import com.github.mizool.typemapping.business.DataType;
+import com.google.common.collect.ImmutableSet;
 
 class JdbcBooleanValueLoadStrategy extends AbstractJdbcValueLoadStrategy
 {
     public JdbcBooleanValueLoadStrategy()
     {
-        super(Types.BOOLEAN, DataType.BOOLEAN);
+        super(ImmutableSet.of(Types.BOOLEAN, Types.BIT), DataType.BOOLEAN);
```

Comment:
this is okay, but `AbstractJdbcValueLoadStrategy` itself (also) needs to ensure that the set is immutable. please replace the `@RequiredArgsConstructor` there with an overloaded constructor for the `Set` case just like the `Integer` one.